docs/user_manual/details.md

```diff
@@ -3,13 +3,13 @@
 Projects, donors, requisitions, cases, and runs each have their own Details page. The Details page
 includes more information about the selected item, all of its cases and QC gates, and the metrics
 for each associated item. Clicking on the name of a project, donor, requisition, or run takes you to
-the corresponding details page. Clicking on an assay name in the cases table links to the Case
-Details page.
+the corresponding details page. Clicking the "Case Details" link in the cases table links to the
+Case Details page.
```

Comment:
I know you modified an existing sentence, but I think it's a little redundant now?

Reply:
It seems so as a sentence, but it's only obvious if you're looking at the cases table. I think it's useful if you're reading this documentation and want to know how to get to the Case Details page.